lib/tests/systems.nix

```
@@ -34,7 +34,7 @@ with lib.systems.doubles; lib.runTests {
   testredox = mseteq redox [ "x86_64-redox" ];
   testgnu = mseteq gnu (linux /* ++ kfreebsd ++ ... */);
   testillumos = mseteq illumos [ "x86_64-solaris" ];
-  testlinux = mseteq linux [ "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "armv7l-linux" "i686-linux" "mips64el-linux" "mipsel-linux" "riscv32-linux" "riscv64-linux" "x86_64-linux" "powerpc64-linux" "powerpc64le-linux" "m68k-linux" "s390-linux" "s390x-linux" "microblaze-linux" "microblazeel-linux" "loongarch64-linux" ];
+  testlinux = mseteq linux [ "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "armv7l-linux" "i686-linux" "loongarch64-linux" "m68k-linux" "microblaze-linux" "microblazeel-linux" "mips-linux" "mips64-linux" "mips64el-linux" "mipsel-linux" "mipsisa32r6-linux" "mipsisa32r6el-linux" "mipsisa64r6-linux" "mipsisa64r6el-linux" "powerpc64-linux" "powerpc64le-linux" "riscv32-linux" "riscv64-linux" "s390-linux" "s390x-linux" "x86_64-linux" ];
```

```suggestion
  testlinux = mseteq linux [
    "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "armv7l-linux"
    "loongarch64-linux" "m68k-linux" "microblaze-linux" "microblazeel-linux"
    "mips-linux" "mips64-linux" "mips64el-linux" "mipsel-linux" "mipsisa32r6-linux" "mipsisa32r6el-linux" "mipsisa64r6-linux" "mipsisa64r6el-linux"
    "powerpc64-linux" "powerpc64le-linux" "riscv32-linux" "riscv64-linux" "s390-linux" "s390x-linux"
    "i686-linux" "x86_64-linux"
  ];
```
that line is a bit very long now. Maybe we can weakly group some things together? Feel free to shuffle things around.